src/platforms/react-native/manual-setup/codepush.mdx

```diff
@@ -70,12 +70,15 @@ sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLO
 ```
 
 For non-macOS devices building for iOS, you will need to pass extra parameters that can't be computed automatically (`--bundle-id` and `--version-name`).
+`--version-name` correspond to `CFBundleShortVersionString` within your `Info.plist`.
```

Comment:
To avoid starting a sentence with text in code format, which is a bit hard to read, I've rearranged it:

```suggestion
Within your `Info.plist`, `--version-name` corresponds to `CFBundleShortVersionString` .
```